ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__serverless-ocp4.15-lp-p2p.yaml

```diff
@@ -0,0 +1,205 @@
+    openshift_eventing-istio-src_knative-v1.12:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-istio-src:knative-v1.12
+    openshift_eventing-kafka-broker-src_knative-v1.12:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-kafka-broker-src:knative-v1.12
+    openshift_knative-eventing-src_knative-v1.12:
+      as:
+      - registry.ci.openshift.org/openshift/knative-eventing-src:knative-v1.12
+    openshift_knative-serving-src_knative-v1.12:
+      as:
+      - registry.ci.openshift.org/openshift/knative-serving-src:knative-v1.12
+  to: serverless-source-image
+- dockerfile_path: serving/ingress/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-8-release-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.16
+  to: serverless-ingress
+- dockerfile_path: serving/metadata-webhook/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-8-release-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.16
+  to: serverless-metadata-webhook
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: serverless-p2p-rosa
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+          [
+            {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+            {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+            {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+            {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+            {"step": "openshift-*", "failure_type": "all", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+         ],
+          "success_rules":
+          [
+            {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
+          ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp-p2p","rosa-classic-lp","serverless-p2p"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      OCM_LOGIN_ENV: staging
```

Comment:
Per our conversation regarding the OpenShift version, it seems the ROSA workflow [uses this environment variable](https://github.com/openshift/release/blob/master/ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-ref.yaml#L53-L55) to get the version:
`OPENSHIFT_VERSION: "4.15"`


[Here is an example.](https://github.com/openshift/release/blob/master/ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-ocp4.14-lp-rosa-classic.yaml#L82)

Reply:
Then we can define the stream with [this](https://github.com/openshift/release/blob/master/ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-ref.yaml#L56-L58), defaults to `stable`.

Reply:
Thanks, updated with `OPENSHIFT_VERSION: "4.15"`